### ISSUE TITLE
Is it correct to set mult_factor in edge parameters when using dropout with pre-training? 

### ISSUE THREAD

**Issue by nian-liu** [nitishsrivastava/deepnet#42]
Hi Nitish,
  In your paper "Improving Neural Networks with Dropout", it says "Dropout nets can also be pretrained using these techniques. The procedure is identical to standard pretraining except with a small modication - the weights obtained from pretraining should be scaled up by a factor of 1/p." Does this mean that I should set mult_factor equal to (1-dropout_prob) in edge parameters in pbtxt files when using dropout with pre-training?
  Another question is that if two layers of an edge have different dropout_prob, for example input layer with dropout_prob 0.2 and hidden1 layer with dropout_prob 0.5, how  to set mult_factor in this condition?
  Thanks in advance.
